From dnax88@gmail.com  Thu Aug 13 16:34:59 2009
From: Daniele Napolitano <dnax88@gmail.com>
Date: Wed, 12 Aug 2009 18:59:21 +0200
Subject: Staging: rt2870: new USB ID for Belkin N+ F5D8055
To: linux-wireless@vger.kernel.org, gregkh@suse.de, akpm@linux-foundation.org, greg@kroah.com
Message-ID: <15d0b9b20908120959v758ed06bl40fb25b6374e2805@mail.gmail.com>


New USB ID for Belkin N+ F5D8055 (USB 802.11n adapter).

I've made this patch with drivers/staging branch found here:
http://git.kernel.org/?p=linux/kernel/git/gregkh/staging-2.6.git

Signed-off-by: Daniele Napolitano <dnax88@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2870/rt2870.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/rt2870/rt2870.h
+++ b/drivers/staging/rt2870/rt2870.h
@@ -102,6 +102,7 @@
 	{USB_DEVICE(0x14B2,0x3C12)}, /* AL */           \
 	{USB_DEVICE(0x050D,0x8053)}, /* Belkin */		\
 	{USB_DEVICE(0x050D,0x815C)}, /* Belkin */		\
+	{USB_DEVICE(0x050D,0x825a)}, /* Belkin */		\
 	{USB_DEVICE(0x14B2,0x3C23)}, /* Airlink */		\
 	{USB_DEVICE(0x14B2,0x3C27)}, /* Airlink */		\
 	{USB_DEVICE(0x07AA,0x002F)}, /* Corega */		\
